frontend/src/routes/(authenticated)/org/[org_id]/OrgMemberTable.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  import { Button } from '$lib/forms';
+  import t from '$lib/i18n';
+  import { Icon } from '$lib/icons';
+  import { createEventDispatcher } from 'svelte';
+  import FormatUserOrgRole from '$lib/components/Orgs/FormatUserOrgRole.svelte';
+  import Dropdown from '$lib/components/Dropdown.svelte';
+  import type { OrgUser, User } from './+page';
+
+  export let shownUsers: OrgUser[];
+
+  const dispatch = createEventDispatcher<{
+    openUserModal: User,
+    changeMemberRole: OrgUser,
+    removeMember: User,
+  }>();
+
+</script>
+
+<div class="overflow-x-auto @container scroll-shadow">
+  <table class="table table-lg">
+    <thead>
+      <tr class="bg-base-200">
+        <th>
+          {$t('admin_dashboard.column_name')}
+          <span class="i-mdi-sort-ascending text-xl align-[-5px] ml-2" />
+        </th>
+        <th>{$t('admin_dashboard.column_email_or_login')}</th>
+        <th class="@2xl:table-cell">
+          {$t('admin_dashboard.column_role')}
+        </th>
+        <th>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      {#each shownUsers as member}
+      {@const user = member.user}
+        <tr>
+          <td>
+            <div class="flex items-center gap-2 max-w-40 @xl:max-w-52">
+              <Button variant="btn-ghost" size="btn-sm" class="max-w-full" on:click={() => dispatch('openUserModal', user)}>
+                <span class="max-width-full overflow-x-clip text-ellipsis" title={user.name}>
+                  {user.name}
+                </span>
+                <Icon icon="i-mdi-card-account-details-outline" />
+              </Button>
+              {#if user.locked}
+                <span
+                    class="tooltip text-warning text-xl leading-0"
+                    data-tip={$t('admin_dashboard.user_is_locked')}>
+                  <Icon icon="i-mdi-lock" />
+                </span>
+              {/if}
+            </div>
+          </td>
+          <td>
+            <span class="inline-flex items-center gap-2 text-left max-w-40">
+              <span class="max-width-full overflow-hidden text-ellipsis" title={user.email ?? user.username}>
+                {user.email ?? user.username}
```

Comment:
this shows 'undefined' when you view the member table as a non admin
![firefox_W4wtZTNYSd](https://github.com/sillsdev/languageforge-lexbox/assets/4575355/b1629d86-8572-444d-8e48-a673bf19bf59). We should probably just hide the column for non admins.

Reply:
Sure. At some point we may decide to expose email addresses to other org members, at which point we may bring the header back. But for now, hiding that column for non-admins (since non-admins don't get to see the information anyway) makes sense.

Reply:
Oh right, non-admins 🤦. Non-admins also shouldn't:
- See the member actions menu
- Be able to click on a user to view their details

Reply:
See https://github.com/sillsdev/languageforge-lexbox/pull/800#issuecomment-2164820236

Reply:
Will do this as part of #881.